README.md

```diff
@@ -9,7 +9,7 @@ Welcome to the 4coder community repository.
 2. call the `package.bat` script from the code directory
    1. `$ cd 4cc\code`
    2. `.\bin\package.bat`
-3. You can also use the `build.bat` script if you want just build the binaries, this does not copy the default config, fonts and bindings
+3. You can also use the `build.bat` script if you want just build the binaries, this does not copy the default config, fonts and bindings, so you'll still still need to call the packaging script first, copy the files under `current_dist...` over to the `build` directory, then run the build script
```

Comment:
This is not true. You don't need to run the packaging script first to do it this way, you can just manually copy the config, fonts and bindings from the  `ship_files` and `dist_files`  folders into the build. Really the build system needs some love and we realistically only need to support one distribution now, there's no reason to keep demo distributions or 32-bit (although might be worth having that one as an option still just in case?)

Reply:
Yeah what you said is true, but does it matter? Why not just run the packaging script and then copy stuff over?
Feel free to edit the commit however you wish, if you have the access permission.

Reply:
TBH i'm just not sure how valuable it is to add this, if you run the `package` script first then there's no reason to call the build script. If its confusing maybe we should just remove the mention of the build script.

Reply:
Wouldn't it be nice to just call the build script to build the binary instead of running the package script every time?

Reply:
No, tbh i basically never build the core, If i need to change something in the core (the extent of which is already committed here) I'll just use the package script then copy it to my working copy is. Otherwise I most of the changes i make are in my own custom layer. Again, really the build system needs reworking, I'd like to get rid of the weird build.cpp stuff too but need to figure out how all the meta-passes fit together. when I get round to that this whole system should get much simpler and the readme will change.

Reply:
Well alright then, I'll leave it to you.